site/scripts/navigation.js

```diff
@@ -0,0 +1,31 @@
+// navigation.js
+
+// Create <nav> element
+const nav = document.createElement("nav");
+
+
+const navList = document.createElement("ul");
+
+// Navigation items
+const navItems = [
+    ["Home", "index.html"],
+    ["Links", "links.html"],
+    ["Glossary", "glossary.html"]
+];
+
+// Loop through items
+navItems.forEach(([text, href]) => {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    a.href = href;
+    a.textContent = text;
+    li.appendChild(a);
+    navList.appendChild(li);
+});
```

Comment:
This could be simplified 😄 
```suggestion
const navItems = [
    {title: "Home", href: "index.html"},
    {title: "Links", href: "links.html"},
    {title: "Glossary", href: "glossary.html"}
];

// Loop through items
navItems.forEach(({title, href}) => {
    const li = document.createElement("li");
    const a = document.createElement("a");
    a.href = href;
    a.textContent = title;
    li.appendChild(a);
    navList.appendChild(li);
});
```